CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -29,6 +30,12 @@
     public DistanceSensor leftDistanceSensor;
     public DistanceSensor rightDistanceSensor;
     
+    public XEncoder leftDriveEncoder;
```

Comment:
No you're totally fine @tfrancisco, I was just commenting that both you and @JohnGilb were adding the same encoders at the same time in two different branches.